### DISCUSSION TITLE
Flow Launcher hard drive results ony when run as an admistrator.

### DISCUSSION THREAD

**Discussion by arbateman** [Flow-Launcher/Flow.Launcher#2576] [Q&A]
When I launch flow launcher with the hotkeys or the icon and look for a file that i know is on my hard drive I get Warning Not all files are Indexed. (I have windows 11 indexing turned off)
When I start flow launcher as an administrator it gives me the proper results.

**Comment by jjw24** [comment]
The 'Warning Not all files are Indexed' comes from PowerToys Run, your alt + space hotkey is still been used by PT Run, not flow.
